tsp_cli_client

```diff
@@ -41,6 +41,7 @@ from tsp.tsp_client import TspClient
 
 TRACE_MISSING = "Trace UUID is missing"
 
+
```

Comment:
extra line or intentional?

edit: Ok, It gets removed in the next commit.

Reply:
yes